src/functTest/projects/groovy-project-compile-config/build.gradle

```diff
@@ -27,6 +27,7 @@ tasks.withType(GroovyCompile) {
 clover {
     compiler {
         encoding = 'UTF-8'
+        additionalGroovycOpts = [listfiles: 'yes']
```

Comment:
Fantastic. This is a good change. Perhaps adding an assertion in the unit test to validate it has done the work?

Reply:
Done.